packages/compass-query-history/src/stores/recent-list-store.js

```diff
@@ -67,6 +78,12 @@ const configureStore = (options = {}) => {
         return r._ns === ns;
       });
 
+      /* Ignore duplicate queries */
+      const existingQuery = this.state.items.find((item) => _.isEqual(comparableQuery(item), recent));
```

Comment:
Looks like we do a similar equality check in this file in `runQuery`: https://github.com/mongodb-js/compass/blob/0ae8aa6f7a2429e7a2081445a490aa43af98f2c1/packages/compass-query-history/src/stores/recent-list-store.js#L102
Would it make sense to share this comparison somehow?

Reply:
I considered this, but then I'm not 100% sure I understand what this line is trying to do. Is it definitely the same as what I wrote above? Then I suppose we should reuse it here. If it is trying to do a looser comparison, then I don't know if that's safe.

Reply:
Actually I suppose that's just tracking, so should be safe enough.